frontend/amundsen_application/static/js/components/Lineage/Graph/chart.spec.ts

```diff
@@ -48,6 +51,18 @@ const nodeWithParent = {
 };
 
 describe('D3 Lineage chart', () => {
+  describe('Generate of shadow node ID should be consistent', () => {
```

Comment:
Nit: The test suite here seems a bit disorganized right now. You could fix it in different ways, but what I would do is something like:
```
 describe('D3 Lineage chart`
     describe('render', ...
        describe('when data is of regular shape'
        describe('when data is X
        describe('when data is Y
     describe('lifetime', ...
        describe('when user clicks in Z
        describe('when updated with W
        ```